test/integration/test.speech_to_text.js

```diff
@@ -380,5 +380,42 @@ describe('speech_to_text_integration', function() {
       speech_to_text.deleteCustomization({ customization_id: customization_id }, done);
       customization_id = null;
     });
+
+    describe('asynchronous api', function() {
```

Comment:
It looks like you accidentally put this inside of the `describe('customization', function() {` section. Just move one of the `});`s from the end of the file to just above here, and then let `npm run autofix` clean up the indentation ;)